### DISCUSSION TITLE
Create custom route in a field

### DISCUSSION THREAD

**Discussion by JustDen1234** [Courseplay/Courseplay_FS22#2524] [Ideas]
I searched for 'Custom route' in issues and discussions, but no results are returned.

Are there plans to add a feature to create custom routes within a field? I'm struggeling creating automatically courses on a map. CP will freeze FS22 trying to calculate a route on a field (map Edgewater Sask), and it would be nice to just drive your own route on a field.

**Comment by Tensuko** [comment]
There won't be recording course in FS22.
Generating a course sometimes needs just patience!
Just becouse it stops you from doing anything doesn't mean the game isn't calculating the course.

**Comment by JustDen1234** [comment]
Thanks for the response. Next time I will be more patient :-p
